modules/nf-core/modules/samtools/stats/main.nf

```diff
@@ -8,7 +8,7 @@ process SAMTOOLS_STATS {
         'quay.io/biocontainers/samtools:1.15.1--h1170115_0' }"
 
     input:
-    tuple val(meta), path(input), path(input_index)
+    tuple val(meta), path(bam), path(bai)
```

Comment:
why are we back to bam/bai here 🙈 since it can also be cram or sam

Reply:
Yes, I notice that, I'll go have a look at this module

Reply:
https://github.com/nf-core/modules/pull/1788/